app/tests/spec/views/mixins/modal-settings-panel-mixin.js

```diff
@@ -0,0 +1,97 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+define([
+  'chai',
+  'sinon',
+  'underscore',
+  'views/mixins/modal-settings-panel-mixin',
+  'views/base',
+  'lib/metrics',
+  'stache!templates/test_template',
+  '../../../lib/helpers'
+], function (Chai, sinon, _, ModalSettingsPanelMixin, BaseView,
```

Comment:
FWIW, it looks like "Chai" is an extreme minority (14 instances of "Chai" vs 352 instances of "chai" in app/tests/**):

```
➜  fxa-content-server git:(master) ✗ grep -rn "chai" app/tests | wc -l
     352

➜  fxa-content-server git:(master) ✗ grep -rn "Chai" app/tests | wc -l
      14

➜  fxa-content-server git:(master) ✗ grep -rn "Chai" app/tests
app/tests/spec/views/mixins/account-locked-mixin.js:20:], function (Chai, Cocktail, sinon, Relier, User, FxaClient, p, AuthErrors,
app/tests/spec/views/mixins/account-locked-mixin.js:24:  var assert = Chai.assert;
app/tests/spec/views/mixins/avatar-mixin.js:21:], function (Chai, sinon, _, AvatarMixin, BaseView, Notifications, Relier,
app/tests/spec/views/mixins/avatar-mixin.js:26:  var assert = Chai.assert;
app/tests/spec/views/mixins/back-mixin.js:13:], function (Cocktail, Chai, sinon, WindowMock,
app/tests/spec/views/mixins/back-mixin.js:17:  var assert = Chai.assert;
app/tests/spec/views/mixins/checkbox-mixin.js:12:], function (Cocktail, Chai, sinon, CheckboxMixin, BaseView, Template) {
app/tests/spec/views/mixins/checkbox-mixin.js:15:  var assert = Chai.assert;
app/tests/spec/views/mixins/loading-mixin.js:13:], function ($, Cocktail, Chai, LoadingMixin, BaseView, Template, WindowMock) {
app/tests/spec/views/mixins/loading-mixin.js:16:  var assert = Chai.assert;
app/tests/spec/views/mixins/service-mixin.js:16:], function (Chai, sinon, _, WindowMock,
app/tests/spec/views/mixins/service-mixin.js:21:  var assert = Chai.assert;
app/tests/spec/views/mixins/timer-mixin.js:11:], function (Chai, _, TestHelpers, TimerMixin, BaseView) {
app/tests/spec/views/mixins/timer-mixin.js:14:  var assert = Chai.assert;
```